core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -1352,6 +1360,28 @@ object GroupMetadataManager {
       "%X".format(BigInt(1, bytes))
   }
 
+  def maybeConvertError(error: Errors) : Errors = {
```

Comment:
nit: can we choose a more descriptive name? Perhaps include `OffsetCommit` somewhere?